.changes/unreleased/tinymce-TINY-12102-2025-05-12.yaml

```
@@ -0,0 +1,7 @@
+project: tinymce
+kind: Fixed
+body: New list item was not inserted correctly, if existing list item had <p> tag
```

Is this wider than `p` does it happen on other blocks like `div` etc?
If so maybe it needs to be: 
```suggestion
body: New list item was not inserted correctly when existing list item had a block element inside.
```